### PR TITLE
eslint rule: id-length

### PR DESCRIPTION
**Pull request by lipis**
http://eslint.org/docs/rules/id-length

### CODE REVIEW

File: app/script/assets/AssetCrypto.js [hunk 1]
```diff
@@ -33,10 +33,10 @@ z.assets.AssetCrypto = (() => {
   function decrypt_aes_asset(ciphertext, key_bytes, reference_sha256) {
     return window.crypto.subtle.digest('SHA-256', ciphertext)
       .then(function(computed_sha256) {
-        const a = new Uint32Array(reference_sha256);
-        const b = new Uint32Array(computed_sha256);
+        const first = new Uint32Array(reference_sha256);
+        const second = new Uint32Array(computed_sha256);
```
Comment (lipis) [inline app/script/assets/AssetCrypto.js]: I'm open for better names here :)

File: app/script/media/MediaEmbeds.js [hunk 2]
```diff
@@ -100,7 +100,7 @@ z.media.MediaEmbeds = (function() {
         .substr(url.indexOf('?'), url.length)
         .replace(/^[?]/, '&')
         .replace(/[&]v=[a-zA-Z0-9_-]{11}/, '')
-        .replace(/[&#]t=([a-z0-9]+)/, (a, b) => `&start=${_convert_youtube_timestamp_to_seconds(b)}`)
+        .replace(/[&#]t=([a-z0-9]+)/, (first, second) => `&start=${_convert_youtube_timestamp_to_seconds(second)}`)
```
Comment (lipis) [inline app/script/media/MediaEmbeds.js]: here as well :)

File: app/script/assets/AssetCrypto.js (hunk 1 above)
Comment (bennycode) [reply app/script/assets/AssetCrypto.js]: Here you are:

```javascript
const typed_reference_sha256 = new Uint32Array(reference_sha256);
const typed_computed_sha256 = new Uint32Array(computed_sha256);
```

File: app/script/media/MediaEmbeds.js (hunk 2 above)
Comment (bennycode) [reply app/script/media/MediaEmbeds.js]: Can you give values for `first` and `second`?

File: app/script/util/ArrayUtil.js [hunk 3]
```diff
@@ -60,8 +60,8 @@ z.util.ArrayUtil = {
       return undefined;
     }
 
-    for (let i = next_index; i <= array.length; i++) {
-      const current_item = array[i];
+    for (let idx = next_index; idx <= array.length; idx++) {
```
Comment (gregor) [inline app/script/util/ArrayUtil.js]: Let's stick with proper names instead of abbreviations

Comment (bennycode) [reply app/script/util/ArrayUtil.js]: Turning `i` into `idx`. 🙈

Comment (lipis) [reply app/script/util/ArrayUtil.js]: It will shadow the `index` that is already used above :D

Comment (lipis) [reply app/script/util/ArrayUtil.js]: the other option is `index_`

Comment (bennycode) [reply app/script/util/ArrayUtil.js]: What about `counter`?  :D

File: app/script/media/MediaEmbeds.js (hunk 2 above)
Comment (lipis) [reply app/script/media/MediaEmbeds.js]: WDYM?

Comment (bennycode) [reply app/script/media/MediaEmbeds.js]: To find a better name for `first` and `second` I need to know what they are (values). :D

Comment (lipis) [reply app/script/media/MediaEmbeds.js]: I guess URL and timestamp.. will use those

**Pull request merged**